src/controller/data_model/BUILD.gn

```diff
@@ -29,7 +29,8 @@ chip_data_model("data_model") {
   allow_circular_includes_from = [ "${chip_root}/src/controller" ]
 }
 
-if (current_os == "android") {
+if (current_os == "android" || getenv("JAVA_PATH") != "") {
+  import("${chip_root}/build/chip/java/rules.gni")
```

Comment:
import should be set the beginning of the build file and should use if (build_java_matter_controller) instead of getenv("JAVA_PATH") != ""

Reply:
no, this file is also used by non-java example app, where java rule cannot be imported

Reply:
I means just move this logic to the beginning of file,  like all include need to be declared at the front of c files, build GN recommends all imports at the beginning of the build gn file

Reply:
done, thanks